docs/contributing_guide.md

```diff
@@ -55,6 +55,8 @@ All pull requests will be reviewed by the JPO-ODE team. The team member will eit
 ### Quarterly Release
 At the end of each quarter, a 'release/version' branch will be created from the develop branch for all release changes. This branch should be used to stabilize the code and prepare for a release. Any non-release changes continue to be pushed to the develop branch as part of the ongoing development process. Validation testing will be conducted on the release branch to ensure that the code is stable and ready for release. Once the release branch is stable, it will be merged into the master branch and tagged with the release version. If hotfixes are necessary during validation, a hotfix branch is created from the master branch. To conclude the release process, changes from the master branch, including all updates and hotfixes, are merged back into the develop branch.
```

Comment:
If we find the need for a hotfix while validating the release branch I believe any hotfix should be made from the release branch not the master branch. These changes would then get populated back to dev once master is merged back into develop later on.

Reply:
I agree. I've adjusted this summary to inform contributors to branch off of the release branch for hotfixes during validation.